packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -76,6 +79,13 @@ type Props = {|
      */
     implicitAllEnabled?: boolean,
 
+    /**
+     * When this is true, the dropdown body shows a search text input at the
+     * top. The items will be filtered by the input.
+     * Selected items will be moved to the top when the dropdown is re-opened.
+     */
+    isFilterable?: boolean,
```

Comment:
ASIDE: I know we haven't done this for other, similar props, so no change requested. 

I wonder if the API would be easier if these were all maybe types; `isFilterable?: ?boolean`

@jandrade?

Reply:
I understand your point. For these cases, where this prop value is statically added (e.g. `isFilterable={true}`, I'd prefer to enforce valid states instead of expecting some truthy or falsy values, but I'm not sure what have been the case with the previous uses of other components. Do you think that we have had less bugs by using maybe types for these flags in WB components?. If that's the case, I think it would be ok to change this to avoid future issues.